text/0000-documentation-related-metadata.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Documentation-related Metadata
+status: DRAFTED
+created_at: 2020-03-11
+updated_at: 2020-03-11
+rfc: https://github.com/salesforce/lwc-rfcs/pull/26
+champion: Nathan McWilliams (@mysticflute) | Lightning Tooling Team
+implementation: 226-228
+---
+
+# Documentation-related Metadata
+
+## Summary
+
+Component authors need a way to provide documentation-related metadata-- metadata that is specifically useful for documentation purposes and not runtime behavior. This includes information like component categorization and supported Salesforce experiences.
+
+This metadata is comprised of key-value pairs and will be put in the documentation markdown file, `docs/module.md`.
+
+## Basic example
+Example documentation markdown file with documentation metadata:
+
+```markdown
+---
+category: Navigation
+experience: Lightning, Communities, Mobile
+---
+
+descriptive documentation...
+```
+
+## Motivation
+
+As part of our path towards enabling customers to document their modules, customers need to be able to add documentation-related metadata. This metadata will be useful for tooling and reference materials. For example, component categories and experiences are displayed in the Component Library documentation website and also used for filtering lists. This metadata can also be used for linting in the LSPs.
+
+In addition, for existing Salesforce-authored modules there's a need to put component authors in control of this metadata. Currently a hardcoded list is maintained by a single team which is a roadblock for other teams creating new components or wishing to update the values.
+
+## Detailed design
+
+Key-value pairs will be placed as [frontmatter](https://github.com/jonschlinkert/gray-matter#what-does-this-do) (in YAML format) in the module markdown file that lives under `__docs__` in the module bundle. For example, this could be the content for `input/__docs__/input.md`:
+
+```markdown
+---
+category: Data Entry
+experience: All
+---
+
+descriptive documentation...
+```
+
+### Recognized Fields
+
+LWC itself will not restrict the set of fields a component author could add to the frontmatter, however only specific fields will be documented as recognized by the Salesforce platform. Unrecognized fields may be present but they will be ignored.
+
+The following keys will be recognized in the frontmatter:
+
+* __category__  `string` \

```

Comment:
It would be up to the consumer to decide which text to use to describe a lack of categorization. None of these fields would be considered required from the compiler standpoint.